src/main/java/us/kilroyrobotics/subsystems/AlgaeIntake.java

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package us.kilroyrobotics.subsystems;
+
+import com.revrobotics.spark.SparkBase.PersistMode;
+import com.revrobotics.spark.SparkBase.ResetMode;
+import com.revrobotics.spark.SparkLowLevel.MotorType;
+import com.revrobotics.spark.SparkMax;
+import com.revrobotics.spark.config.SparkMaxConfig;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import us.kilroyrobotics.Constants.AlgaeConstants;
+
+public class AlgaeIntake extends SubsystemBase {
+    private SparkMax algaeMotorLeader;
+    private SparkMax algaeMotorFollower;
+
+    /** Creates a new AlgaeIntake. */
+    public AlgaeIntake() {
+        this.algaeMotorLeader =
+                new SparkMax(AlgaeConstants.kAlgaeMotorLeaderId, MotorType.kBrushed);
```

Comment:
Brushed motor? What motor are we using

Reply:
We are using CIMs temporarily. We don't know what we are using for final product. Maybe the two neo 550s we have?